.github/workflows/deploy.yml

```diff
@@ -4,11 +4,82 @@ on:
   push:
     branches:
       - master
+  pull_request:
```

Comment:
It will be confusing to deploy on PR right now. Ideally PR should build its own separate environment (review apps).
Let's keep it simple and only deploy on push to master for now.